bigtable/conformance_test.sh

```diff
@@ -49,11 +48,12 @@ function cleanup() {
 }
 trap cleanup EXIT
 
-pushd $conformanceTestsHome > /dev/null;
-    # Run the conformance tests
-    go test -v -proxy_addr=:$testProxyPort | tee -a $sponge_log
-    RETURN_CODE=$?
-popd > /dev/null;
+# Run the conformance tests
+cd $conformanceTestsHome
+go install golang.org/dl/go1.20.2@latest
+go1.20.2 download
```

Comment:
Should these 2 lines above be removed?

Reply:
Conformance tests can only be run on Go 1.20.2. This is very specific to Bigtable. So, installation is being done here and not the common conformance.yaml

Reply:
Nit: Can you add a comment explaining *why* Go 1.20.2 is required? We will want to know for when we inevitably bump the minimum version to Go 1.21 in N years.

Reply:
Added